### ISSUE TITLE
Add 'make glyphlist from font' feature, use Noto fonts to make a Noto collection

### ISSUE THREAD

**Issue by davelab6** [davelab6/pyfontaine#51]
After #46 is resolved....
 
The Noto fonts - git repo at http://code.google.com/p/noto - offer a large collection of different writing systems, with carefully curated character and glyph lists. It would be great to compare other fonts to those.

- [x] Add a 'compare glyphs' feature which makes a charmap/glyphlist on the fly from the first font and compares subsequent fonts to it. 
- [x] Add a 'make glyphlist from font' feature that outputs a glyphlist from the glyphs present in a given font
- [x] Add a 'make charmap from font' feature as above
- [x] Download and run each script over all the Noto fonts to make 2 Noto collections, `noto-chars` and `noto-glyphs`, and check them in

Eg

```sh
$ pyfontaine --compare-chars font1.ttf font2.ttf;
$ pyfontaine --compare-glyphs font1.ttf font2.ttf;
$ pyfontaine --new-charmap-py NotoSans-Regular.ttf  > notosans-regular.py;
$ pyfontaine --new-glyphmap-py NotoTamilSans-Regular.ttf > nototamilsans-regular.py;
$ pyfontaine --new-charmap-txt NotoSans-Regular.ttf  > notosans-regular.txt;
$ pyfontaine --new-glyphmap-txt NotoTamilSans-Regular.ttf > nototamilsans-regular.txt;

```

**Comment by vitalyvolkov** [comment]
I have checked functional that already done. It seems that now what left is 

> Download and run each script over all the Noto fonts to make 2 Noto collections, noto and noto, and check them in

In this check you write noto and noto :) but what exactly noto should be run over on?

**Comment by davelab6** [comment]
See example; the 2 collections are glyphs and chars based

**Comment by vitalyvolkov** [comment]
Where to upload generated python files for Noto?

**Comment by davelab6** [comment]
Same directory as others

**Comment by vitalyvolkov** [comment]
As I understand we then will have about 244 files for one noto font. Is that right?

**Comment by davelab6** [comment]
Which noto font is that?

**Comment by vitalyvolkov** [comment]
```
vitaly@vitaly-X550LB:~/Documents/Dave/noto$ ls -lah fonts/individual/hinted/*.ttf | wc -l
94
vitaly@vitaly-X550LB:~/Documents/Dave/noto$ ls -lah fonts/individual/unhinted/*.ttf | wc -l
155
```

**Comment by davelab6** [comment]
Put them in a noto subfodler? :)

**Comment by vitalyvolkov** [comment]
Please check the https://github.com/davelab6/pyfontaine/commit/ef7020d7ef961ea6bc9dc00d799747756794f9a4

**Comment by davelab6** [comment]
Great!
